Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/github-issue-2563.dfy

```diff
@@ -1,2 +1,2 @@
-// RUN: %exits-with 1 %dafny /compile:0 '/verificationLogger:csv;log.csv' "%s" > "%t"
```

Comment:
Could we leave in both lines to have some assurance that they produce the same results?

Reply:
They don't produce the same results. You can see from the diff in the expect file that the error message is slightly different. There's already various small differences in reporting from the new and old CLI, and generally we only test the new CLI in that case. If you want I can test both here but it's a complicated topic.